docs/backend/embedding_model.ipynb

```diff
@@ -46,43 +46,43 @@
      "text": [
```

Comment:
Rewrite this following the style of this notebook https://sgl-project.github.io/backend/openai_api_vision.html
Name: OpenAI APIs - Embedding